_data/simple-icons.json

```
@@ -11386,6 +11386,11 @@
                 "type": "CC-BY-SA-4.0"
             }
         },
+        {
+            "title": "OsmAnd",
+            "hex": "FF8800",
+            "source": "https://github.com/osmandapp/OsmAnd-misc/tree/master/logo/osmand"
```

```suggestion
            "source": "https://github.com/osmandapp/OsmAnd-misc/blob/9ec3bacebf580d0862ded5813a4aa934d0862302/logo/osmand/symbol_osmand.svg"
```

Let's use the permalink.